app/Http/Middleware/Authenticate.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Illuminate\Auth\Middleware\Authenticate as Middleware;
+
+class Authenticate extends Middleware
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing doc comment for class Authenticate](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)